mediagrains/tools/wrap_in_gsf.py

```diff
@@ -144,10 +146,13 @@ def wrap_audio_in_gsf():
     flow_id = args.flow_id if args.flow_id else uuid.uuid4()
     source_id = args.source_id if args.source_id else uuid.uuid4()
 
+    duration = fractions.Fraction(args.samples_per_grain, args.sample_rate)
```

Comment:
This needs to be in the `if args.format == CogAudioFormat.AAC:` scope below because the other case provides defaults.